tests/metrics/test_regression.py

```diff
@@ -7,12 +7,13 @@
 import dask_ml.metrics
 
 
-@pytest.fixture(params=["mean_squared_error", "mean_absolute_error", "r2_score"])
+@pytest.fixture(params=["mean_squared_error", "mean_absolute_error", "mean_absolute_percentage_error", "r2_score"])
```

Comment:
```suggestion
@pytest.fixture(
    params=[
        "mean_squared_error",
        "mean_absolute_error",
        "mean_absolute_percentage_error",
        "r2_score",
    ]
)
```

Looks like `black==19.10b0` [isn't happy](https://dev.azure.com/dask-dev/dask/_build/results?buildId=2037&view=logs&j=73591ef5-68f9-550f-ee9a-e10f200815d9&t=28d49b26-095c-5073-6ee0-65d189ec82f7&l=21) about the line length here.

Reply:
Also, perhaps, it'd be nice if the correctness of the method was sanity-checked against its sklearn counterpart, just as it's done for some of the other metrics a bit further down in the same test file.

Reply:
> Looks like black==19.10b0 isn't happy about the line length here.

Ah ok. If `dask-ml` has chosen to pin to older versions of linters, then I think the non-conda option documented at https://ml.dask.org/contributing.html#style will be unreliable, since https://github.com/dask/dask-ml/blob/f5e5bb4d4d21782b0e93f0f6541e6f2501b0c06c/setup.py#L29 doesn't have a pin for things like `black`.

Once i switched to the conda instructions there, I got the expected diff. Updated in https://github.com/dask/dask-ml/pull/822/commits/1142fcc93a95e2af72211410959afed14116121d.

> Also, perhaps, it'd be nice if the correctness of the method was sanity-checked against its sklearn counterpart, just as it's done for some of the other metrics a bit further down in the same test file.

Can you clarify what you want me to change? As far as I can tell, that is exactly what happens by adding `mean_squared_percentage_error` to the `metric_pairs` fixture. Every metric in that fixture is tested against its scikit-learn equivalent by https://github.com/dask/dask-ml/blob/f5e5bb4d4d21782b0e93f0f6541e6f2501b0c06c/tests/metrics/test_regression.py#L37.

Reply:
> Ah ok. If dask-ml has chosen to pin to older versions of linters, then I think the non-conda option documented at https://ml.dask.org/contributing.html#style will be unreliable

You're absolutely right! I've got a PR over at https://github.com/dask/dask-ml/pull/813 waiting to be reviewed (for a couple of weeks now), and subsequently merged. It should improve the static-checking situation.

> Every metric in that fixture is tested against its scikit-learn equivalent by

Indeed - ignore me about this one, please! I got confused that we could introduce extra tests like the `test_mean_squared_log_error` one.

Reply:
I'll bring up the question of whether the `setup.py` versions of the linters should be pinned, too, in https://github.com/dask/dask-ml/pull/813.